invenio_app_ils/records/api.py

```diff
@@ -38,11 +41,34 @@
 class IlsRecord(Record):
     """Ils record class."""
 
+    @property
+    def pid(self):
+        """Get the PersistentIdentifier for this record."""
+        return PersistentIdentifier.get(
+            pid_type=self._pid_type,
+            pid_value=self[self.pid_field],
+        )
+
+    @staticmethod
+    def pid_type_to_record_class(pid_type):
+        """Get the record class from the pid_type."""
+        if pid_type == "loanid":
+            endpoints_cfg = current_app.config["CIRCULATION_REST_ENDPOINTS"]
+        else:
+            endpoints_cfg = current_app.config["RECORDS_REST_ENDPOINTS"]
```

Comment:
I think you could just rely on the current_app.config["RECORDS_REST_ENDPOINTS"] as [here](https://github.com/inveniosoftware/invenio-circulation/blob/master/invenio_circulation/ext.py#L46) we are merging the 2 configs :)